<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefaultActions</class>
 <widget class="QWidget" name="DefaultActions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>374</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string notr="true">DeaDBeeF</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/root/images/deadbeef.png</normaloff>:/root/images/deadbeef.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>348</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>Playback</string>
    </property>
    <widget class="QMenu" name="loopingMenu">
     <property name="title">
      <string>Repeat</string>
     </property>
     <property name="icon">
      <iconset theme="media-playlist-repeat">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <addaction name="actionLoopAll"/>
     <addaction name="actionLoopTrack"/>
     <addaction name="actionLoopNothing"/>
    </widget>
    <widget class="QMenu" name="shuffleMenu">
     <property name="title">
      <string>Shuffle</string>
     </property>
     <property name="icon">
      <iconset theme="media-playlist-shuffle">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <addaction name="actionNoShuffle"/>
     <addaction name="actionTrackShuffle"/>
     <addaction name="actionAlbumShuffle"/>
     <addaction name="actionRandomTrackShuffle"/>
    </widget>
    <addaction name="shuffleMenu"/>
    <addaction name="loopingMenu"/>
    <addaction name="actionScrollPlayback"/>
    <addaction name="actionCursorPlayback"/>
    <addaction name="actionStopTrack"/>
    <addaction name="actionStopAlbum"/>
    <addaction name="separator"/>
    <addaction name="actionJump_to_current_track"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFiles"/>
    <addaction name="separator"/>
    <addaction name="actionAddFiles"/>
    <addaction name="actionAddFolder"/>
    <addaction name="actionAddURL"/>
    <addaction name="actionAddAudioCD"/>
    <addaction name="separator"/>
    <addaction name="actionNewPlaylist"/>
    <addaction name="actionLoadPlaylist"/>
    <addaction name="actionSaveAsPlaylist"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuSelection">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Selection</string>
     </property>
     <property name="icon">
      <iconset theme="edit-select">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <addaction name="actionSelectionRemove"/>
     <addaction name="actionSelectionCrop"/>
    </widget>
    <widget class="QMenu" name="menuSort_By">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Sort by</string>
     </property>
     <property name="icon">
      <iconset theme="view-sort-descending">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <addaction name="actionSortTitle"/>
     <addaction name="actionSortTrackNumber"/>
     <addaction name="actionSortAlbum"/>
     <addaction name="actionSortArtist"/>
     <addaction name="actionSortDate"/>
     <addaction name="actionSortRandom"/>
     <addaction name="actionSortCustom"/>
    </widget>
    <addaction name="actionClearAll"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionDeselectAll"/>
    <addaction name="actionInvert_selection"/>
    <addaction name="menuSelection"/>
    <addaction name="actionFind"/>
    <addaction name="menuSort_By"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionBlockToolbarChanges"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuPlayback"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAddFolder">
   <property name="icon">
    <iconset theme="folder-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Add folder(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNoShuffle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="actionTrackShuffle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tracks</string>
   </property>
  </action>
  <action name="actionAlbumShuffle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Albums</string>
   </property>
  </action>
  <action name="actionLoopAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>All Tracks</string>
   </property>
  </action>
  <action name="actionLoopTrack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>One Track</string>
   </property>
  </action>
  <action name="actionLoopNothing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="gtk-preferences">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+P</string>
   </property>
  </action>
  <action name="actionAddFiles">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Add file(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAddAudioCD">
   <property name="icon">
    <iconset theme="media-optical-audio">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Add Audio CD</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionAddURL">
   <property name="icon">
    <iconset theme="folder-remote">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Add location</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+U</string>
   </property>
  </action>
  <action name="actionSaveAsPlaylist">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save playlist</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoadPlaylist">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Load playlist</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+L</string>
   </property>
  </action>
  <action name="actionNewPlaylist">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New playlist</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+N</string>
   </property>
  </action>
  <action name="actionBlockToolbarChanges">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Design mode</string>
   </property>
  </action>
  <action name="actionClearAll">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-clear">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-select-all">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+A</string>
   </property>
  </action>
  <action name="actionDeselectAll">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-select-none">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Deselect all</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionFind">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-find">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionOpenFiles">
   <property name="icon">
    <iconset theme="list-add">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open file(s)...</string>
   </property>
  </action>
  <action name="actionRandomTrackShuffle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Random Tracks</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionScrollPlayback">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Scroll follows playback</string>
   </property>
  </action>
  <action name="actionCursorPlayback">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cursor follows playback</string>
   </property>
  </action>
  <action name="actionStopAlbum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stop after current album</string>
   </property>
  </action>
  <action name="actionJump_to_current_track">
   <property name="icon">
    <iconset theme="go-jump">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Jump to current track</string>
   </property>
  </action>
  <action name="actionInvert_selection">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="edit-select-invert">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Invert selection</string>
   </property>
  </action>
  <action name="actionSelectionCrop">
   <property name="icon">
    <iconset theme="transform-crop">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionSelectionRemove">
   <property name="icon">
    <iconset theme="edit-delete-remove">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionSortTitle">
   <property name="icon">
    <iconset theme="view-media-title">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </action>
  <action name="actionSortTrackNumber">
   <property name="icon">
    <iconset theme="view-media-track">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Track number</string>
   </property>
  </action>
  <action name="actionSortArtist">
   <property name="icon">
    <iconset theme="view-media-artist">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Artist</string>
   </property>
  </action>
  <action name="actionSortAlbum">
   <property name="icon">
    <iconset theme="view-media-album-cover">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Album</string>
   </property>
  </action>
  <action name="actionSortDate">
   <property name="icon">
    <iconset theme="view-calendar">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </action>
  <action name="actionSortRandom">
   <property name="icon">
    <iconset theme="randomize">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Random</string>
   </property>
  </action>
  <action name="actionSortCustom">
   <property name="icon">
    <iconset theme="edit-entry">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Custom</string>
   </property>
  </action>
  <action name="actionStopTrack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stop after current track</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>onAboutAction()</slot>
  <slot>onAboutQtAction()</slot>
 </slots>
</ui>
